src/pages-and-resources/discussions/app-config-form/apps/shared/discussion-topics/TopicItem.jsx

```diff
@@ -102,17 +102,33 @@ const TopicItem = ({
             >
               <Collapsible.Visible whenClosed>
                 {getHeading(false)}
-                <div className="py-2 ml-auto">
-                  <ExpandMore />
+                <div className="ml-auto">
+                  <IconButton
+                    alt="Expand"
```

Comment:
Please check if we can translate this text.

Reply:
yes done